common/autoconf/toolchain_windows.m4

```diff
@@ -25,7 +25,7 @@
 
 ################################################################################
 # The order of these defines the priority by which we try to find them.
-VALID_VS_VERSIONS="2010 2012 2013 2015 2017"
+VALID_VS_VERSIONS="2013 2015 2012 2010 2017 2019"
```

Comment:
Order should be: `2010 2012 2013 2015 2017 2019`

Reply:
2017 (build) and 2019 (configuration). To me it's enough at this stage, the changes do not touch older toolkits, only define new 2019. Moreover it's a bit difficult to test for now, because 5-6 more backports required to enable VS2019, and few of them require complete regression. For now VS2019 build will fail immediately

Reply:
Fixed. The patch tested for all 2010-2019 toolkits, the PR header was updated accordingly

Reply:
Ping